encrypted-media-respec.html

```diff
@@ -3023,17 +3027,17 @@
         <ul>
           <li>
             <p>
-              Access network resources, either local or remote, except via the user agent, or as part of the user agent, as explicitly permitted by this specification.
+              Access network resources, either local or remote, except via the user agent as explicitly permitted by this specification.
```

Comment:
I agree that such implementations are allowed and that "via the user agent" includes this.

As I said in #411, the CDM is always _logically_ distinct from the user agent regardless of how they are implemented, so I don't think this is necessary. Since it increases the complexity and decreases the clarity of what is intended to be a clarifying statement, I recommend we remove it.